index.html

```diff
@@ -18,9 +18,9 @@
 		<!-- Header -->
 			<section id="header">
 				<header>
-					<span class="image avatar"><img src="images/avatar.jpg" alt="" /></span>
+					<!-- <span class="image avatar"><img src="images/avatar.jpg" alt="" /></span> -->

```

Comment:
can we not delete it instead f commenting out? If we need to go back we have `git` :)